sql/testdata/builtin_function

```diff
@@ -258,13 +258,75 @@ query error syntax error at or near "\("
 SELECT current_date()
 
 query T
-SELECT now() - current_timestamp < interval '10s'
+SELECT now() - transaction_date::timestamp < interval '24h10s'
```

Comment:
Other tests for `now()` exist in `testdata/datetime`. Perhaps put these new datetime function tests there as well.